tensorflow/core/framework/memory_types.cc

```diff
@@ -50,6 +50,8 @@ void MemoryTypesHelper(const NameRangeMap& name_map,
   for (size_t i = 0; i < host_memory_args->size(); ++i) {
     auto iter = name_map.find((*host_memory_args)[i]);
     if (iter != name_map.end()) {
+      VLOG(2) << (*host_memory_args)[i] << " " << iter->second.first << " "
```

Comment:
Are these vlogs going to make any sense if someone hasn't carefully read the code? Probably want to make them more verbose if we keep them.